### DISCUSSION TITLE
OR-tools crashes for 100 nodes CVRP problem

### DISCUSSION THREAD

**Discussion by rrlcs** [google/or-tools#3815] [Routing (and legacy CP) questions]
**What version of OR-Tools and what language are you using?**
Version: ortools 9.6.2534
Language: Python

**Linux**

I'm using it to solve CVRP with 100 nodes. It crashes and dmesg gives following message:
Unstable LPIB (352800 >= 176400); disabling LPIB delay counting

What are the limitations of OR-tools?

**Comment by lperron** [comment]
None. And I do not know what lbip is.

**Comment by Mizux** [comment]
AFAIK `LPIB` is not part of or-tools
```sh
$ git grep -in "lpib"
```

Otherwith the limitation of or-tools is the limitation of the host machine RAM (node indices are encoded as int64).

**Reply by rrlcs** [reply]
yes lpib is not part of or-tools but that's the message I get from kernel.

Is 100 nodes too much for a 16 gb RAM system?

**Comment by lperron** [comment]
no.
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00

**Reply by rrlcs** [reply]
In that case, what could be the reason that it is giving solutions for 50 nodes but abruptly exits the program in case of 100 nodes?

**Comment by lperron** [comment]
wrong data, node used more than once. SetRange() called with incompatible
values...

Most likely an error in your code.
Laurent Perron | Operations Research | ***@***.*** | (33) 1 42 68 53
00

**Reply by rrlcs** [reply]
Ok thanks a lot for the help.
